### PR TITLE
Add style to the components

### PR DESCRIPTION
**Pull request by alvp01**
In this feature I styled the whole web app using only scss.

### CODE REVIEW

File: src/app.scss [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+.align-center {
+  align-items: center;
+}
+
+.container {
+  width: 100vw;
+  height: 100vh;
+  background-color: #f5f6fa;
+}
```
Comment (julie-ify) [inline src/app.scss]: - [x] Good job with the design so far. However, I noticed that the heading of the app disappears when I added up to two books. After checking your CSS styling, I found out that the issue is caused by `height: 100vh;` given to the `.container`. Let's fix this issue by removing the `height: 100vh;` or making the `height: 100%;` so as to improve the UI of your app. 

When there is one book:
Loom_qdr3K3stiI

When there are more than one book:
Loom_u29Zo4fCVN

File: src/components/Book.jsx [hunk 2]
```diff
@@ -1,34 +1,48 @@
...
         <h4 className="book-author">{author}</h4>
-        <div className="options">
-          <button type="button">Comments</button>
-          <button type="button" onClick={handleClick}>Remove</button>
-          <button type="button">Edit</button>
+        <div className="d-flex flex-row">
+          <button type="button" className="btn-link">Comments</button>
+          <button type="button" onClick={handleClick} className="btn-link">Remove</button>
+          <button type="button" className="btn-link">Edit</button>
         </div>
```
Comment (julie-ify) [inline src/components/Book.jsx]: - [x] You have done a good job so far. However, these elements have a right border in the Zeplin design, please let's add it to make sure our design matches that of Zeplin.

Your Implementation | Zeplin Design
:---:|:---:
 |

File: src/components/Book.jsx [hunk 3]
```diff
@@ -1,34 +1,48 @@
...
-        <div className="current-chapter">Current Chapter</div>
-        <div className="book-chapter">{currentChapter}</div>
-        <button type="button">Update progress</button>
+      <div className="book-progress d-flex flex-row">
+        <div className="progress d-flex flex-row">
+          <AiOutlineLoading3Quarters />
+          <div className="completion-container">
+            <p className="completion-number">
+              {`${progressPercentage}%`}
+            </p>
```
Comment (julie-ify) [inline src/components/Book.jsx:42]: - [x] The circular progress bar looks a little bit different from the design. Let's make it look like a real progress bar by making the radius clear.See highlight below:

Your Implementation | Zeplin Design
:---:|:---:
 |

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @alvp01, 

Good job so far!

## Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Good use of SCSS ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
## Hi @alvp01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alvp01** [comment]
> ## Hi @alvp01, 
> 
> Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
> Congratulations! 🎉 
> 
> 
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> 
> ------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you for your time and work! Happy coding

**Pull request merged**